msgid ""
msgstr ""
"Project-Id-Version: openplotter\n"
"POT-Creation-Date: 2021-01-17 13:13+0100\n"
"PO-Revision-Date: 2021-03-04 18:52\n"
"Last-Translator: \n"
"Language-Team: Italian\n"
"Language: it\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Generator: Poedit 1.8.7.1\n"
"X-Poedit-Basepath: ../../..\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Poedit-SourceCharset: UTF-8\n"
"X-Poedit-SearchPath-0: .\n"
"X-Crowdin-Project: openplotter\n"
"X-Crowdin-Project-ID: 289888\n"
"X-Crowdin-Language: it\n"
"X-Crowdin-File: openplotter-network.po\n"
"X-Crowdin-File-ID: 14\n"

#: networkPostInstall.py:30
msgid "Installing python packages..."
msgstr "Installazione pacchetti python..."

#: networkPostInstall.py:33 networkPostInstall.py:51 networkPostInstall.py:63
#: networkPostInstall.py:69 networkPreUninstall.py:31 networkPreUninstall.py:62
msgid "DONE"
msgstr "FATTO"

#: networkPostInstall.py:34 networkPostInstall.py:52 networkPostInstall.py:64
#: networkPostInstall.py:70 networkPreUninstall.py:32 networkPreUninstall.py:57
#: networkPreUninstall.py:63
msgid "FAILED: "
msgstr "FALLITO: "

#: networkPostInstall.py:37
msgid "Editing config files..."
msgstr "Modifica file di configurazione..."

#: networkPostInstall.py:55
msgid "Enabling services..."
msgstr ""

#: networkPostInstall.py:66
msgid "Setting version..."
msgstr "Impostazione versione..."

#: networkPreUninstall.py:28
msgid "Removing python packages..."
msgstr "Rimozione pacchetti python..."

#: networkPreUninstall.py:35
msgid "Restoring network Debian defaults..."
msgstr "Ripristino della rete predefinita Debian..."

#: networkPreUninstall.py:48
msgid "Removing openplotter-network service..."
msgstr "Rimozione del servizio openplotter-network..."

#: networkPreUninstall.py:56
msgid "DONE. PLEASE REBOOT TO RESTORE NETWORK SETTINGS"
msgstr "FATTO. ESEGUIRE RIAVVIO PER RIPRISTINARE IMPOSTAZIONI DI RETE"

#: networkPreUninstall.py:59
msgid "Removing version..."
msgstr "Rimozione versione..."

#: openplotterNetwork.py:38
msgid "Network"
msgstr "Rete"

#: openplotterNetwork.py:48
msgid "Help"
msgstr "Guida"

#: openplotterNetwork.py:51
msgid "Settings"
msgstr "Impostazioni"

#: openplotterNetwork.py:54
msgid "Addresses and Ports"
msgstr "Indirizzi e porte"

#: openplotterNetwork.py:56
msgid "Check Network"
msgstr "Controlla la Rete"

#: openplotterNetwork.py:59
msgid "Install Wifi Drivers"
msgstr "Installa Driver WiFi"

#: openplotterNetwork.py:59
msgid "This does only help for unrecognized usb wlan type:"
msgstr "Questo aiuta solo per gli adattatori usb WiFi non riconosciuti:"

#: openplotterNetwork.py:66
msgid "Access Point"
msgstr "Access Point"

#: openplotterNetwork.py:134
msgid "Network Mode"
msgstr "Modalità Rete"

#: openplotterNetwork.py:139
msgid "AP"
msgstr "AP"

#: openplotterNetwork.py:149
msgid "5 GHz"
msgstr "5GHz"

#: openplotterNetwork.py:151
msgid "Add Ethernet port to the AP"
msgstr ""

#: openplotterNetwork.py:174
msgid "Access Point Settings"
msgstr "Impostazioni Access Point"

#: openplotterNetwork.py:177
msgid "Sharing Internet device"
msgstr "Condivisione dispositivo Internet"

#: openplotterNetwork.py:185
msgid "SSID \n"
"maximum 32 characters"
msgstr "SSID\n"
"massimo 32 caratteri"

#: openplotterNetwork.py:193
msgid "Password \n"
"minimum 8 characters required"
msgstr "Password\n"
"minimo 8 caratteri"

#: openplotterNetwork.py:202
msgid "Channel"
msgstr "Canale"

#: openplotterNetwork.py:312 openplotterNetwork.py:332
#: openplotterNetwork.py:337 openplotterNetwork.py:387
#: openplotterNetwork.py:391 openplotterNetwork.py:433
#: openplotterNetwork.py:466 openplotterNetwork.py:484
#: openplotterNetwork.py:614
msgid "none"
msgstr "nessuno"

#: openplotterNetwork.py:404
msgid "Wait please... "
msgstr "Attendere... "

#: openplotterNetwork.py:538
msgid "Your SSID must have a maximum of 32 characters and your password a minimum of 8."
msgstr "Il tuo SSID deve avere un massimo di 32 caratteri e la password di un minimo di 8."

#: openplotterNetwork.py:589
msgid "OpenPlotter will reboot. If something goes wrong and you are on a headless system, you may not be able to reconnect again.\n\n"
"Are you sure?"
msgstr "OpenPlotter si riavvierà. Se qualcosa va storto e sei su un sistema senza monitor, potresti non essere in grado di riconnetterti di nuovo.\n\n"
"Sei sicuro?"

#: openplotterNetwork.py:590
msgid "Question"
msgstr "Domanda"

#: openplotterNetwork.py:613 openplotterNetwork.py:614
msgid "wifi access point: "
msgstr "wifi access point: "

#: openplotterNetwork.py:631
msgid "wifi client: "
msgstr "wifi client: "

#: openplotterNetwork.py:644
msgid "running"
msgstr "in esecuzione"

#: openplotterNetwork.py:645
msgid "stopped"
msgstr "arrestato"

#: openplotterNetwork.py:659
msgid "IP address\n"
msgstr "Indirizzo IP\n"

#: openplotterNetwork.py:672
msgid "Installing Wifi modules... "
msgstr "Installazione moduli WiFi... "

#: openplotterNetwork.py:674
msgid "Done."
msgstr "Fatto."

#: openplotterNetwork.py:712
msgid "There are conflicts between server connections"
msgstr "Ci sono conflitti tra le connessioni server"

#: openplotterNetwork.py:713
msgid "No conflicts between server connections"
msgstr ""

#: ports.py:28
msgid "VNC Remote Desktop"
msgstr "Desktop remoto VNC"

#: startup.py:46
msgid "Checking WIFI Access Point password..."
msgstr "Verifica password WIFI Access Point..."

#: startup.py:68
msgid "Security warning: You are using the default WIFI Access Point password.\n"
"Please change password in OpenPlotter Network."
msgstr "Avviso di sicurezza: stai utilizzando la password di default per l'Access Point WIFI.\n"
"Cambia la password in OpenPlotter Network."

#: startup.py:69
msgid "changed"
msgstr "modificato"

